robot/resources/lib/python_keywords/http_gate.py

```diff
@@ -352,3 +351,12 @@ def try_to_get_object_via_passed_request_and_expect_error(
     except Exception as err:
         match = error_pattern.casefold() in str(err).casefold()
         assert match, f"Expected {err} to match {error_pattern}"
+
+
+@allure.step("Return bearer token in base64 to caller")
```

Comment:
looks like it can be put to `acl.py` module

Reply:
done